### PR TITLE
fix(AAE-1012): Unreleased Liquibase lock prevents the application start up

### PR DESCRIPTION
**Pull request by igdianov**
This PR adds support for running Liquibase database module inside deployment init container for the following applications:

- [x] activiti-cloud-query
- [x] activiti-cloud-modeling 

Depends on https://github.com/Activiti/activiti-cloud/pull/246
Part of Activiti/Activiti#3595
Part of https://alfresco.atlassian.net/browse/AAE-1012

### CODE REVIEW

**Review by erdemedeiros — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -51,13 +55,17 @@ jobs:
...
+    script: make docker/activiti-cloud-modeling-liquibase
+
   - name: Build Activiti Cloud Modeling
     stage: docker
     script: make docker/activiti-cloud-modeling
 
   - name: Run Acceptance Tests
     stage: test
-    after_script: make delete || echo "Trying to delete helm chart"
+    after_script: echo make delete || echo "Trying to delete helm chart"
```
Comment (mteodori) [inline .travis.yml]: is this a testing leftover?

File: activiti-cloud-query/pom.xml [hunk 2]
```diff
@@ -118,6 +118,9 @@
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
```
Comment (mteodori) [inline activiti-cloud-query/pom.xml:121]: why this?

**Comment by mteodori** [comment]
we can use the liquibase image directly https://www.liquibase.org/blog/using-liquibase-in-kubernetes

File: activiti-cloud-query/pom.xml (hunk 2 above)
Comment (igdianov) [reply activiti-cloud-query/pom.xml:121]: It blows up on Windows when trying to exec `mvn spring-boot:run`

File: .travis.yml (hunk 1 above)
Comment (igdianov) [reply .travis.yml]: It is temporary to be able to inspect Helm release in k8s.

**Review by erdemedeiros — APPROVED** [review]

**Pull request merged**